scripts/006-gcc-newlib-SPU.sh

```diff
@@ -1,22 +1,25 @@
 #!/bin/sh -e
+#
 # gcc-newlib-SPU.sh by Naomi Peori (naomi@peori.ca)
+# Modified by luizfernandonb (luizfernando.nb@outlook.com)
 
 GCC="gcc-7.2.0"
 NEWLIB="newlib-1.20.0"
+PS3="-PS3"
 
```

Comment:
Don't get it why we need an extra variable here? These scripts only build for PS3, so no need imo, to add another "indirection" here.

Reply:
Just to make it easier to do maintenance if you ever need to change the script

Reply:
i still dont see the advantage. All you'll have to change here is probably the version of newlib and gcc. The "PS3" is only used for the patch scripts which is fully under our control.